src/common/components/forms/CancelSellForm.tsx

```diff
@@ -51,7 +51,7 @@ const CancelSellForm: FC<CancelSellFormProps> = ({
       toast(`Canceling listing for ${nft.name}`);
       await sdk
         .transaction()
-        .add(sdk.listings(listing.auctionHouse).cancel({ listing, nft }))
+        .add(sdk.listings(listing.auctionHouse!).cancel({ listing, nft }))
```

Comment:
Why the `!`?

Reply:
It is because in AhListing type auctionHouse is optional. So, here we were getting error that auctionHouse might be undefined.

Reply:
Is it easier to add a if(listing.auctionHouse) above?

Reply:
Yes, that's possible too. Let me do that.

Reply:
But we also have another option to remove optional from the auctionHouse, as in all listings/offers/purchases we will definitely have the auctionHouse. @kespinola should we remove optional for auctionHouse in the indexer?